mri/operators/__init__.py

```diff
@@ -17,3 +17,7 @@
 # Linear operators
 from .linear.wavelet import WaveletN, WaveletUD2
 from .linear.dictionary import DictionaryLearning
+
+# Gradient operators
```

Comment:
Again, I think the import names are self explanatory, I am sorry I let that slip the first time, but I don't think we need those extra comments (see this [SO answer](https://stackoverflow.com/a/7533090/4332585) for example).

Reply:
Will remove them